src/test/java/pages/BasePage.java

```diff
@@ -428,5 +438,83 @@ protected boolean isAlertDisplayed() {
             return false;
         }
     }
+
+    protected String getSaltString10() {
+        String SALTCHARS = "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz";
```

Comment:
specify name of the string SALTCHARS? what does it means

Reply:
I don`t understand what you want)

Reply:
SALTCHARS it means that there is different letters

Reply:
@KristinaPereselkina check new PR)

Reply:
resolved